From f4c38233cef941faed2302cd2d5e51969516e532 Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Fri, 9 Oct 2015 14:41:10 +0800
Subject: [PATCH 0807/5242] MLK-11681-1 ARM: imx: enable sata on imx6qp sdb
 board

commit  e5f737457142f04980dae3144cdd0c7859fb98ce from
https://source.codeaurora.org/external/imx/linux-imx.git

enable sata support on imx6qp sdb board.

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
(cherry picked from commit 8b08d0e657a121136eebe5bb6a998b7c9a6bff25)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index f62b1ab..61691ddd 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -147,3 +147,7 @@
 	memory-region = <&memory>;
 	status = "okay";
 };
+
+&sata {
+	status = "okay";
+};
-- 
1.7.9.5

